src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -29,7 +29,6 @@
 				class="conversations-search"
 				:is-searching="isSearching"
 				@input="debounceFetchSearchResults"
-				@submit="onInputEnter"
```

Comment:
https://github.com/nextcloud/spreed/commit/8c34922084f92b1cdfbe01352ae145ca7f441571

I just fixed this recently?
Did you solve a conflict wrong or why do you remove that hitting enter opens the first conversation?

Reply:
oh, this is something I found while testing: there was no handler for "onInputEnter" so I removed this line

I've double checked now and just realized that the handler is hidden in a mixin... okay, I'll revert this

Reply:
reverted